python/dries_json_reader.py

```diff
@@ -48,6 +48,75 @@ def cells_json(data):
     return cell_dict
 
 
+def cell_sets_json(data):
+    '''
+    >>> data = {
+    ...     "cell_1": {
+    ...         "locations": [1632.02, -1305.7],
+    ...         "factors": {
+    ...             "pleiden_clus": [3],
+    ...             "kmeans": [8]
+    ...         },
+    ...         "mappings": {
+    ...             "tsne": [-11.0776, 6.0311],
+    ...             "umap": [-6.858, 15.7691]
+    ...         }
+    ...     }
+    ... }
+    >>> cell_sets = cell_sets_json(data)
+    >>> list(cell_sets.keys())
+    ['version', 'datatype', 'tree']
+    >>> cell_sets['datatype']
+    'cell'
+    >>> len(cell_sets['tree'])
+    2
+    >>> sorted([ n['name'] for n in cell_sets['tree'] ])
+    ['Leiden Clustering', 'k-means Clustering']
+    '''
+    clustering_dict = {
+        'pleiden_clus': {},
+        'kmeans': {}
+    }
+    nice_names = {
+        'pleiden_clus': 'Leiden Clustering',
```

Comment:
This is just for our demos... but I would steer clear of making things prettier than what the data can provide. If the demos are ugly, that's some pressure on the labs to give us better data.

Reply:
Right, I think it is OK for the demo data here.